The removed test works in most cases but seems to fail on some architectures.
Difficult to reproduce as it seems the failures are dependent on if the system is
overloaded.
https://github.com/flux-framework/flux-sched/issues/1384

diff -rupN flux-sched-0.45.0.orig/t/CMakeLists.txt flux-sched-0.45.0/t/CMakeLists.txt
--- flux-sched-0.45.0.orig/t/CMakeLists.txt	2025-05-08 22:18:30.000000000 +0200
+++ flux-sched-0.45.0/t/CMakeLists.txt	2025-07-28 14:31:52.788564856 +0200
@@ -18,7 +18,6 @@ set(ALL_TESTS
   t1013-qmanager-priority.t
   t1014-annotation.t
   t1015-find-format.t
-  t1016-nest-namespace.t
   t1017-rv1-bootstrap.t
   t1018-rv1-bootstrap2.t
   t1019-qmanager-async.t
